libobs/graphics/graphics.c

```diff
@@ -28,6 +28,11 @@
 #include "effect-parser.h"
 #include "effect.h"
 
+#ifdef _MSC_VER
+#undef near
+#undef far
+#endif
```

Comment:
From the other PR:
> There are some symbol conflicts in VS and it's preventing the whole thing from compiling.
> Changing them is only a temporary measure to get it working.

How temporary?  Why do I not get these conflicts when I compile for Win32 or x64?

Reply:
I did some rebasing work on mentioned PRs, and I merged them back to this branch before the rebasing. 
This should be no longer temporary since it's the best solution so far.

The reason you don't get these conflicts is unclear. My speculation is there are something to do with CMake's generated `.sln` for Windows ARM64. To prove that I did the same thing but targeting x64, it compiled just fine.

I was wondering the same thing as you, why would it work on x64 build? `near` and `far` are "reserved" macros in `minwindef.h` since 16-bit era Windows, and it's still there for backward compatibility reason. We have the same code, same headers, the only difference will be the object file we are linking to.
This is the `minwindef.h` from my machine:
![image](https://user-images.githubusercontent.com/7903172/120061956-3a567680-c025-11eb-9e1a-24e4e3ee855c.png)

Reference:
https://devblogs.microsoft.com/oldnewthing/20200728-00/?p=104012